.github/workflows/ci.yml

```diff
@@ -0,0 +1,68 @@
+name: "ci"
+
+on:
+  pull_request:
+    branches:
+      - "main"
+  workflow_dispatch:
+
+jobs:
+  Spec:
+    uses: "puppetlabs/cat-github-actions/.github/workflows/module_ci.yml@main"
+    secrets: "inherit"
+
+  Integration:
```

Comment:
could we have a reusable `Integration` test workflow?

Reply:
Technically yeah, but this is the only module that it would apply to so wouldn't have any real benefit :-)